src/css/main.css

```diff
@@ -7,6 +7,8 @@
 @import "./js_interop.css";
 @import "./tooltips.css";
 
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Source+Sans+Pro:wght@400;600;700;900&display=swap");
```

Comment:
I don't think we need the Roboto font, no? And for sens pro only weight 600 would be enough. That said, we can try the logo with text version I sent over Slack first :)

Reply:
I pruned it in a separate commit but a logo with the text would be even better :)